### PR TITLE
Gokhan_Branch5 Pull request

### PR DESCRIPTION
**Pull request by GokhanRepo**
Added files to Branch5
Please review!

### CODE REVIEW

**Review by matthewstbbs — APPROVED** [review]
awesome at 20221214, 2338

File: src/files/NumbersToArray.java [hunk 1]
```diff
@@ -14,6 +14,8 @@ public static void main(String[] args) {
         System.out.println("Numbers stored : " + Arrays.toString(numbers));
     }
```
Comment (GulsumOztomsuk) [inline src/files/NumbersToArray.java:15]: //System.out.println("Numbers stored : " + numbers);  --> error

**Review by GulsumOztomsuk — APPROVED** [review]

**Review by evadamtk — COMMENTED** [review]
ok

File: src/files/NumbersToArray.java (hunk 1 above)
Comment (GokhanRepo) [reply src/files/NumbersToArray.java:15]: I tapped on "Resolve conversation"

**Pull request merged**